CHANGES.rst

```diff
@@ -1,3 +1,8 @@
+3.0
+===
+
+Changed to using dictionaries instead of parsing XML.
```

Comment:
A  backward-incompatible release should provide more explanation and context for an existing user of the library to understand what actions they need to take to adopt this new version.